microsite/docusaurus.config.js

```diff
@@ -204,7 +204,10 @@ module.exports = {
           {
             items: [
               {
-                html: '<a href="/"><h2 class="footerLogo"></h2></a>',
+                html: `
+                <a href="/" aria-label="Backstage Software Catalog and Developer Platform">
```

Comment:
Since this link takes you back to home of the microsite (meaning its a functional image), I'd recommend describing the purpose a bit more. E.g. `Backstage Home`. See for example the [Example 1: Image used alone as a linked logo](https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo)